android-exoplayer/src/main/java/com/brentvatne/entity/RNTranslations.java

```diff
@@ -0,0 +1,104 @@
+package com.brentvatne.entity;
+
+import androidx.annotation.NonNull;
+
+import java.util.Map;
+
+public class RNTranslations {
+
+    private static final String KEY_EPG_LABEL = "player_epg_button";
+    private static final String KEY_STATS_LABEL = "player_stats_button";
+    private static final String KEY_PLAY_LABEL = "player_play_button";
+    private static final String KEY_PAUSE_LABEL = "player_pause_button";
+    private static final String KEY_AUDIO_AND_SUBTITLES_LABEL = "player_audio_and_subtitles_button";
+    private static final String KEY_LIVE_LABEL = "live";
+    private static final String KEY_FAVORITE_LABEL = "favourite";
+    private static final String KEY_MORE_VIDEOS_LABEL = "moreVideos";
+    private static final String KEY_WATCH_LIST_LABEL = "watchlist";
+
+    private static final String DEFAULT_EPG_LABEL = "Schedule";
+    private static final String DEFAULT_STATS_LABEL = "Stats";
+    private static final String DEFAULT_PLAY_LABEL = "Play";
+    private static final String DEFAULT_PAUSE_LABEL = "Pause";
+    private static final String DEFAULT_AUDIO_AND_SUBTITLES_LABEL = "Audio & Subtitles";
+    private static final String DEFAULT_LIVE_LABEL = "Live";
+    private static final String DEFAULT_FAVORITE_LABEL = "Favourite";
+    private static final String DEFAULT_MORE_VIDEOS_LABEL = "More Videos";
+    private static final String DEFAULT_WATCH_LIST_LABEL = "Watch List";
+
+    private final String epgLabel;
+    private final String statsLabel;
+    private final String playLabel;
+    private final String pauseLabel;
+    private final String audioAndSubtitlesLabel;
+    private final String liveLabel;
+    private final String favoriteLabel;
+    private final String moreVideosLabel;
+    private final String watchListLabel;
+
+    public RNTranslations(@NonNull Map<String, Object> translations) {
+        this.epgLabel = translations.get(KEY_EPG_LABEL) != null ?
+                        (String) translations.get(KEY_EPG_LABEL) :
```

Comment:
so I think we could make this DRYER by creating a method to handle setting a translation because this code feels verbose and repetitive